src/vma/util/agent.cpp

```diff
@@ -82,12 +86,14 @@ agent* g_p_agent = NULL;
 
 agent::agent() :
 		m_state(AGENT_CLOSED), m_sock_fd(-1), m_pid_fd(-1),
-		m_msg_num(512), m_msg_grow(16)
+		m_msg_num(AGENT_DEFAULT_MSG_NUM), m_msg_grow(AGENT_DEFAULT_MSG_GROW),
+		m_inactive_treshold(AGENT_DEFAULT_INACTIVE), m_alive_treshold(AGENT_DEFAULT_ALIVE)
```

Comment:
Why do we need to add m_msg_grow/m_inactive_treshold/m_alive_treshold as a class members ? cant we use the "defines" instead ?

Reply:
it is a implementation style that allows to introduce configuration of this parameters easier if it is needed in the future.

Reply:
i understand. but still, I do not think we should use class members for constant variables.
It is very easy to add class members while its needed...